translations/harbour-berlin-vegan-nl.ts

```diff
@@ -194,97 +201,153 @@
         <translation>Restaurant</translation>
     </message>
     <message id="id-venue-subtype-fastfood">
-        <location filename="../qml/pages/VenueFilterSettings.qml" line="142"/>
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="16"/>
         <source>Snack bar</source>
         <translation>Snackbar</translation>
     </message>
     <message id="id-venue-subtype-cafe">
-        <location filename="../qml/pages/VenueFilterSettings.qml" line="152"/>
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="21"/>
         <source>Café</source>
         <translation>Café</translation>
     </message>
     <message id="id-venue-subtype-icecream">
-        <location filename="../qml/pages/VenueFilterSettings.qml" line="162"/>
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="26"/>
         <source>Ice cream parlor</source>
-        <translation>IJssalon</translation>
+        <translation>Ijssalon</translation>
+    </message>
+    <message id="id-venue-subtype-foods">
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="40"/>
+        <source>Food</source>
+        <translation>Eten</translation>
+    </message>
+    <message id="id-venue-subtype-clothing">
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="45"/>
+        <source>Fashion</source>
+        <translation>Kleding</translation>
+    </message>
+    <message id="id-venue-subtype-toiletries">
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="50"/>
+        <source>Toiletries</source>
+        <translation>Toiletartikelen</translation>
+    </message>
+    <message id="id-venue-subtype-supermarket">
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="55"/>
+        <source>Supermarket</source>
+        <translation>Supermarket</translation>
+    </message>
+    <message id="id-venue-subtype-hairdressers">
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="60"/>
+        <source>Hairdresser&apos;s</source>
+        <translation>Kappers</translation>
+    </message>
+    <message id="id-venue-subtype-sports">
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="65"/>
+        <source>Sports</source>
+        <translation>Sport</translation>
+    </message>
+    <message id="id-venue-subtype-tattoostudio">
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="70"/>
+        <source>Tattoo studio</source>
+        <translation>Tattoo studio</translation>
+    </message>
+    <message id="id-venue-subtype-accommodation">
+        <location filename="../qml/components-generic/VenueSubTypeDefinitions.qml" line="75"/>
+        <source>Accommodation</source>
+        <translation>Accommodatie</translation>
```

Comment:
@Vistaus do you like to check my changes for the Dutch translation?

Reply:
Done! Corrected a few things, LGTM now :)